SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -64,5 +64,26 @@ public async Task<IActionResult> CreateArticleAsync([FromBody] CreateArticleDTO
 
             return BadRequest("Unable to create article.");
         }
+
+        [Authorize(Roles = "Admin")]
+        [HttpPut("EditArticle")]
+        public async Task<IActionResult> EditArticleAsync([FromBody] CreateArticleDTO adminInput)
+        {
+            ValidationResult validationResult = await _articleValidator.ValidateAsync(adminInput);
+            if (!validationResult.IsValid)
+            {
+                var response = _generateModelStateDictionary.modelStateDictionary(validationResult);
+                return ValidationProblem(response);
+            }
+
+            bool editedSuccessfylly = await _articleService.EditArticle(adminInput);
+
+            if (editedSuccessfylly)
+            {
+                return Ok("Article updated successfully.");
```

Comment:
Please do not use magic strings.

Reply:
Object result messages are now in ValidationMessages.